### PR TITLE
Update copyright year to 2023

### LINKED ISSUE #275: Please copyright for 2023
Make sure to update copyright for 2023

❤️

### PR DESCRIPTION
**Pull request by jesse-peters**
Closes #275

### CODE REVIEW

**Comment by krisnova** [comment]
Unfortunately we will need to change the string from:

```
Apache 2.0 License Copyright © 2022 The Aurae Authors 
```

to a range

```
Apache 2.0 License Copyright © 2022-2023 The Aurae Authors 
```

Also can we please update the copyright on the website in this PR as well?

**Comment by jesse-peters** [comment]
@krisnova would you mind giving this a check when you have a moment

File: .github/workflows/031-ubuntu-aurae-builder-image-ubuntu-docker-build-tester.yml [hunk 1]
```diff
@@ -35,7 +35,7 @@ permissions:
  packages: write
 env:
  CARGO_TERM_COLOR: always
- CACHE_VERSION: v1
+ CACHE_VERSION: v2
```
Comment (future-highway) [inline .github/workflows/031-ubuntu-aurae-builder-image-ubuntu-docker-build-tester.yml:38]: Is this an intentional change?

File: Cargo.toml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # ---------------------------------------------------------------------------- #
-#             Apache 2.0 License Copyright © 2022 The Aurae Authors            #
+#             Apache 2.0 License Copyright © 2023 The Aurae Authors            #
```
Comment (future-highway) [inline Cargo.toml:2]: missed this one

Comment (jesse-peters) [reply Cargo.toml]: Thanks, I found 2 actually :)

File: .github/workflows/031-ubuntu-aurae-builder-image-ubuntu-docker-build-tester.yml (hunk 1 above)
Comment (jesse-peters) [reply .github/workflows/031-ubuntu-aurae-builder-image-ubuntu-docker-build-tester.yml:38]: Yea. I was working on splitting the dockerfile builds at the same time as this PR, and that PR overwrote the image this run needed. Flipping this forced it to use a new clean image.

**Comment by jesse-peters** [comment]
@future-highway should be set now. I want to get this one in today if possible, because its going to need to be updated pretty constantly :)

**Review by future-highway — APPROVED** [review]

**Pull request merged**